docs/tempo/website/metrics-generator/_index.md

```diff
@@ -1,19 +1,19 @@
 ---
-title: Server-side metrics
+aliases:
+- /docs/tempo/latest/server_side_metrics/
+title: Metrics-generator
 weight: 400
 ---
 
-# Server-side metrics
+# Metrics-generator
 
-Server-side metrics is a feature that derives metrics from ingested traces.
-
-To generate metrics, it uses an additional component: the **metrics-generator**.
+Metrics-generator is an optional Tempo component that derives metrics from ingested traces. 
```

Comment:
Should metrics-generator be bolded the way it was in the line that was removed? 
```suggestion
The **metrics-generator** is an optional Tempo component that derives metrics from ingested traces. 
```

Reply:
I don't think bolded is necessary. It was originally bolded to emphasise the name of the component since the page is called _server-side metrics_. Now the whole page/section is called metrics-generator, it should be obvious enough 🙂

Reply:
Sounds good.